price-feeder/oracle/oracle.go

```diff
@@ -460,22 +522,22 @@ func (o *Oracle) tick(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	oracleParams, err := o.GetParams()
+
+	blockHeight, err := rpcclient.GetChainHeight(clientCtx)
 	if err != nil {
 		return err
 	}
-	if err := o.SetPrices(ctx, oracleParams.AcceptList); err != nil {
-		return err
+	if blockHeight < 1 {
+		return fmt.Errorf("expected positive block height")
 	}
 
-	o.checkAcceptList(oracleParams)
-
-	blockHeight, err := rpcclient.GetChainHeight(clientCtx)
+	oracleParams, err := o.GetCachedParams(uint64(blockHeight))
```

Comment:
Why don't we store cached block height as an int64? that way we don't have to convert anything here
```suggestion
	oracleParams, err := o.GetCachedParams(blockHeight)
```

Reply:
to be able to use [here](https://github.com/umee-network/umee/blob/c5a846e4a6ccf7646532d4313d4a9274f6884cd3/price-feeder/oracle/oracle.go#L399)

Reply:
You should just be able to use o.oracleOnChain.params == oracletypes.Params{} 👍